core/blockly.js

```diff
@@ -141,36 +141,30 @@ Blockly.svgSize = function(svg) {
 };
 
 /**
- * Schedule a call to the resize handler.  Groups of simultaneous events (e.g.
- * a tree of blocks being deleted) are merged into one call.
- * @param {Blockly.WorkspaceSvg} workspace Any workspace in the SVG.
+ * Size the workspace when the contents change.  This also updates
+ * scrollbars accordingly.
+ * @param {!Blockly.WorkspaceSvg} opt_workspace. The svg group of the workspace
```

Comment:
This param is optional, so remove the !.  SVG should be uppercase in the comment.